### PR TITLE
Creating random user-agent to fix broken streams

### PR DESCRIPTION
**Pull request by fimdomeio**
This introduces a random generated string that is used as the user agent.

Should fix most problems reported in #51

### CODE REVIEW

File: tv.sh [hunk 1]
```diff
@@ -51,6 +51,10 @@ __tvi() { echo "https://video-auth6.iol.pt/live_tvi/live_tvi/playlist.m3u8?wmsAu
 __tvi24() { echo "https://video-auth6.iol.pt/live_tvi24/live_tvi24/playlist.m3u8?wmsAuthSign=$(wget https://services.iol.pt/matrix?userId= -o /dev/null -O -)"; }
 __euronews() { echo $(wget http:$(wget http://pt.euronews.com/api/watchlive.json -O - -o /dev/null | cut -d\" -f4 | sed 's/\\//g') -O - -o /dev/null | cut -d\" -f12 | sed 's/\\//g'); }
 
+#random string for user-agent to avoid blacklists
+USER_AGENT=$(head /dev/urandom | tr -dc A-Za-z0-9 | head -c 50)
+echo $USER_AGENT
```
Comment (marado) [inline tv.sh:56]: We don't really need this echo around...

**Review by marado — CHANGES_REQUESTED** [review]
That extra echo can be removed

**Comment by fimdomeio** [comment]
fixed

**Review by marado — APPROVED** [review]

**Review by marmelo — APPROVED** [review]

**Comment by marmelo** [comment]
Thanks @fimdomeio, @marado!
Sorry for the delay.

**Pull request merged**